src/test/regress/expected/citus_local_tables.out

```diff
@@ -485,10 +485,175 @@ NOTICE:  executing the command locally: TRUNCATE TABLE citus_local_tables_test_s
+ t
+(1 row)
+
+  SELECT count(*) FROM pg_locks where relation='citus_local_table_4'::regclass;
+ count
+---------------------------------------------------------------------
+     1
+(1 row)
+
+COMMIT;
+-- hide first column (relationId) as it might change
+SELECT part_storage_type, part_method, part_key, part_replica_count, part_max_size, part_placement_policy FROM master_get_table_metadata('citus_local_table_4');
+ part_storage_type | part_method | part_key | part_replica_count | part_max_size | part_placement_policy
+---------------------------------------------------------------------
+ t                 | n           |          |                  1 |       1536000 |                     2
+(1 row)
+
+SELECT master_get_table_ddl_events('citus_local_table_4');
+                                        master_get_table_ddl_events
+---------------------------------------------------------------------
+ CREATE TABLE citus_local_tables_test_schema.citus_local_table_4 (a integer)
+ ALTER TABLE citus_local_tables_test_schema.citus_local_table_4 OWNER TO postgres
+ CREATE INDEX citus_local_table_4_idx ON citus_local_tables_test_schema.citus_local_table_4 USING btree (a)
+(3 rows)
+
+SELECT column_to_column_name(logicalrelid, partkey)
+FROM pg_dist_partition WHERE logicalrelid = 'citus_local_table_4'::regclass;
+ column_to_column_name
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT column_name_to_column('citus_local_table_4', 'a');
+                                                  column_name_to_column
+---------------------------------------------------------------------
+ {VAR :varno 1 :varattno 1 :vartype 23 :vartypmod -1 :varcollid 0 :varlevelsup 0 :varnoold 1 :varoattno 1 :location -1}
+(1 row)
+
+SELECT master_update_shard_statistics(shardid)
+FROM (SELECT shardid FROM pg_dist_shard WHERE logicalrelid='citus_local_table_4'::regclass) as shardid;
+ master_update_shard_statistics
+---------------------------------------------------------------------
+                           8192
+(1 row)
+
+-- will always be no-op as we create the shell table from scratch
+-- while creating a citus local table, but let's see it works
+SELECT truncate_local_data_after_distributing_table('citus_local_table_4');
+ truncate_local_data_after_distributing_table
+---------------------------------------------------------------------
+
+(1 row)
+
+BEGIN;
+  SELECT worker_drop_distributed_table('citus_local_table_4');
+ worker_drop_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+  -- should only see shard relation
+  SELECT tableName FROM pg_catalog.pg_tables WHERE tablename LIKE 'citus_local_table_4%';
+          tablename
+---------------------------------------------------------------------
+ citus_local_table_4_1504037
+(1 row)
+
+ROLLBACK;
+-- should return a single element array that for its own shard id
```

Comment:
could change the last part of the sentence as `that for its own shard id` seems broken